service/src/main/java/org/cbioportal/genome_nexus/service/transformer/ExternalResourceTransformer.java

```diff
@@ -50,11 +53,8 @@ private List<T> mapJsonToInstance(DBObject rawJson, Class<T> type, ObjectMapper
     {
         List<T> list = new ArrayList<>();
         ObjectMapper mapper = objectMapper;
-
-        if (mapper == null)
-        {
-            mapper = new ObjectMapper();
-        }
+        mapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```

Comment:
I need to add this because there is an enum mapping issue:
`"Cannot deserialize value of type `org.oncokb.client.Query$ReferenceGenomeEnum` from String "GRCh37": not one of the values accepted for Enum class: [GRCH37, GRCH38] at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: org.oncokb.client.IndicatorQueryResp["query"]>org.oncokb.client.Query["referenceGenome"])"`